src/OTP/TOTP.php

```diff
@@ -36,6 +36,32 @@ protected function getDefaultCounter()
         return time();
     }
 
+    /**
+     * Gets current Time Step value
+     *
+     * @return int
+     */
+    public function getTimeStep()
+    {
+        return $this->timeStep;
+    }
+
+    /**
+     * Sets Time Step value
+     *
+     * @param int $step
+     * @return int
+     * @throws Exception
+     */
+    public function setTimeStep($step)
+    {
+        if (! is_int($step) || ! $step > 0) {
```

Comment:
I didn't configure a style linter, but would like to keep it PSR-12 compatible, so could we format this differently?
Also I don't thing _not_ and _greater than_ should be combined because is simpler to use _lower than_
```suggestion
        if (!is_int($step) || $step < 1) {
```